### ISSUE TITLE
Callback not called if path is invalid

### ISSUE THREAD

**Issue by bencergazda** [zmxv/react-native-sound#359]
When the file doesn't exist on the given (absolute) path

```javascript
var woosh = new Sound('invalid path', null, (error) => {
  if (error) {
    console.log('failed to load the sound', error)
    return
  }  // loaded successfully
  console.log('duration in seconds: ' + this.player.getDuration() + 'number of channels: ' + this.player.getNumberOfChannels()) // DEBUG
});
```

Expected result would be a message in the console, saying `failed to load the sound ...`, but nothing like this happens.

    "react-native": "0.49.3",
    "react-native-sound": "0.10.5"

Tested on Android API Level 27 emulator and Level 21 device

**Comment by bencergazda** [comment]
Sorry, looks like the duplicate of #305

**Comment by paulmelnikow** [comment]
Fixed in #405!
